Lib/http/cookies.py

```diff
@@ -605,8 +605,15 @@ class SimpleCookie(BaseCookie):
     received from HTTP are kept as strings.
     """
     def value_decode(self, val):
+        """
+        Return an unquoted string, from the cookie header in a reversed fasion of value_encode.
```

Comment:
To be consistent with other method documentation, could
you start write the docstring on the first line, please?

Reply:
Done for both, thanks!